gm4_crossbow_cartridges/data/gm4_crossbow_cartridges/functions/torch/torch_place.mcfunction

```diff
@@ -3,8 +3,8 @@
 # at @s
 
 #place torch
-execute if block ~.05 ~ ~ #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:wall_torch[facing=west]
-execute if block ~ ~ ~.05 #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:wall_torch[facing=north]
-execute if block ~-.05 ~ ~ #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:wall_torch[facing=east]
-execute if block ~ ~ ~-.05 #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:wall_torch[facing=south]
-execute if block ~ ~-.05 ~ #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:torch
+execute if block ~.05 ~ ~ #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:wall_torch[facing=west] destroy
+execute if block ~ ~ ~.05 #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:wall_torch[facing=north] destroy
+execute if block ~-.05 ~ ~ #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:wall_torch[facing=east] destroy
+execute if block ~ ~ ~-.05 #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:wall_torch[facing=south] destroy
+execute if block ~ ~-.05 ~ #gm4_crossbow_cartridges:torch_placeable run setblock ~ ~ ~ minecraft:torch destroy
```

Comment:
I think you may be able to duplicate torches when you fire the arrow into a corner of two blocks

Reply:
You're right, was wondering why I didn't do this before but that explains it

Reply:
@misode Cant dupe it, just tested 👍 (in corner)

Reply:
You could dupe it. It checks for a block ~0.05 next to it which is extremely small but it is possible